src/game/GameProcMain.cpp

```diff
@@ -4173,7 +4173,7 @@ bool CGameProcMain::MsgSend_NPCEvent(short siIDTarget) {
     int  iOffset = 0;
 
     CAPISocket::MP_AddByte(byBuff, iOffset, N3_WARP_LIST);
-    CAPISocket::MP_AddByte(byBuff, iOffset, WI.iID); // ���� ���̵� ������...
+    CAPISocket::MP_AddShort(byBuff, iOffset, WI.iID); // ���� ���̵� ������...
```

Comment:
Why which from byte to short here?

Reply:
As you can see below:
https://github.com/srmeier/KnightOnline/blob/80268e2fa971389a3e94c430966a7943c2631dbf/Client/WarFare/GameProcMain.cpp#L4205C15-L4205C24
https://discord.com/channels/385253339971190794/1004036880444162089/1130637972795039805